core/raftlease/store.go

```diff
@@ -345,8 +354,8 @@ func RecoverError(resp *ResponseError) error {
 	switch resp.Code {
 	case "invalid":
 		return lease.ErrInvalid
-	case "concurrent-update":
```

Comment:
Is this something that can bite us with different agent versions expecting different strings?
Or are we ok because this is only controller <-> controller and those are kept at the same version?

Reply:
Yes; controller-only concern here.